src/Processors/Formats/Impl/ArrowColumnToCHColumn.cpp

```diff
@@ -272,7 +272,7 @@ static ColumnPtr readByteMapFromArrowColumn(std::shared_ptr<arrow::ChunkedArray>
         for (size_t value_i = 0; value_i != static_cast<size_t>(chunk->length()); ++value_i)
             bytemap_data.emplace_back(chunk->IsNull(value_i));
     }
-    return nullmap_column;
+    return std::move(nullmap_column);
```

Comment:
`std::move` doesn't produce any effect in return statement. Or I'm wrong?

Reply:
I add std::move because vscode clangd remind me that return value optimization is not enabled if `nullmap_column` 'has different type with `ColumnPtr`.

Reply:
Ok

Reply:
I found the  words in https://en.cppreference.com/w/cpp/language/copy_elision
```
In a return statement, when the operand is the name of a non-volatile object with automatic storage duration, which isn't a function parameter or a catch clause parameter, and which is of the same class type (ignoring cv-qualification) as the function return type. This variant of copy elision is known as NRVO, "named return value optimization".
```